src/shared/custom-rpc/getPayloadId.ts

```diff
@@ -0,0 +1,9 @@
+let i = 1;
+const initialValue = crypto.getRandomValues(new Uint32Array(1))[0];
+
+/**
+ * returns a unique random value always larger than the previous one
```

Comment:
May be this can be rephrased, cause numbers are not random any more, only the base is random. However, not critical